utils/build.ps1

```diff
@@ -2471,6 +2529,10 @@ if ($Clean) {
   }
 }
 
+if (-not $SkipBuild -and $IncludeDS2) {
+  Invoke-BuildStep Build-RegsGen2 $HostArch
+}
```

Comment:
Why not hoist this into the tools build above? I think it makes sense to build this around the compilers phase and in the same block.